vehicles/src/modules/special-auth/dto/request/upsert-special-auth.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { AutoMap } from '@automapper/classes';
+import { ApiProperty } from '@nestjs/swagger';
+
+import { IsOptional } from 'class-validator';
+import { NoFeeType } from 'src/common/enum/no-fee-type.enum';
+
+export class UpsertSpecialAuthDto {
```

Comment:
Update

Reply:
It works for both insert and update scenario hence the name. do you still think update is better suited as per REST conventions, if so then I will change the name to include update instead of upsert.

Reply:
Please change to Update as per OnRouteBC standards